apps/example-app/src/app/home/components/default-phone-layout/default-phone-layout.component.scss

```diff
@@ -1,26 +1,8 @@
-/* Phone Module */
-section.phone {
-    /* CSS Variables for device size */
-    /* Needs to be set here in order to apply the % to the right module */
-    /* If we set this properties at the top with the other ones, the % will apply to the full viewport instead the module */
-    /* This are called local scope variables */
-    --device-height: 70vh;
-    --device-width: 15vw;
-    /* This height and width are relatives to the viewport */
-    right: 22vw;
-    top: 14vh;
-    z-index: 2;
-    display: flex;
-    align-items: center;
-    justify-content: flex-end;
-    position: absolute;
-}
-div.phone-wrapper {
+.phone-wrapper {
```

Comment:
By reducing the number of elements we use, we dramatically decrease the amount of code we need to write and maintain 

cc @Fdom92 @PaulPaulDevelops @Nachzz